lib/rack/tracker/handler.rb

```diff
@@ -56,4 +56,28 @@ def write_event(event)
   def handler_name
     self.class.name.demodulize.underscore
   end
+
+  def tracker_options
+    @_tracker_options ||= {}.tap do |tracker_options|
+      options.slice(*self.class::ALLOWED_TRACKER_OPTIONS).each do |key, value|
+        if option_value = value.respond_to?(:call) ? value.call(env) : value
+          tracker_options[tracker_option_key(key)] = tracker_option_value(option_value)
+        end
+      end
+    end
+  end
+
+  private
+
+  # Transformations to be applied to tracker option keys.
+  # Override in descendants, if necessary.
+  def tracker_option_key(key)
+    key.to_sym
+  end
+
+  # Transformations to be applied to tracker option values.
+  # Override in descendants, if necessary.
+  def tracker_option_value(value)
+    value.to_s
```

Comment:
why do we need the `to_s`? 
If we use it in the template the `<%= %>` does a `to_s` anyway, isn't it?

Reply:
That's what the Google Analytics tracker is currently doing. It can potentially make a difference in that case, because the values aren't outputted directly using the `<%= %>` there, but rather the entire hash is converted to JSON, so it's a difference between `"{\"one\":1}"` and `"{\"one\":\"1\"}"`. That said, it probably shouldn't be the default behaviour, I can move it to the GA tracker.

Reply:
ok, I see. good point. 
I guess the convention could be that `call` should return a string. but I am fine either way. 
If that's the case right now in the analytics tracker then let's do it like that.